### DISCUSSION TITLE
Marp extension for Zed

### DISCUSSION THREAD

**Discussion by VivaldoMendes** [marp-team/marp#546] [Ideas]
Hi, first, let me congratulate the developers of Marp. I have been using it for two years, and what a fantastic and fast way to write down slides! I have used it with VSCode, a good but somewhat slow editor.

I have been playing with Zed recently, and it is much, much faster than VSCode. Will a Marp extension for Zed be considered?

Thank you very much.

**Comment by yhatt** [comment]
Currently we have no plan for Zed integration, but [our core](https://github.com/marp-team/marp-core) is an open-end library. If interested, developer can try to develop the extension for Zed, as like as [integrations for other editors by Marp community](https://github.com/marp-team/awesome-marp#community). :)

**Comment by VivaldoMendes** [comment]
Thanks for the quick reply. I would certainly do it if I had the knowledge. However, I am a user, and unfortunately, I do not have that knowledge. 

But I will undoubtedly continue to use Marp in VScode. Marp is a beautiful piece of software.

**Comment by RoyiAvital** [comment]
@yhatt , What about integration into [Quarto](https://github.com/quarto-dev/quarto-cli/)?

**Remark**: I am not able to open new discussion, on search this was the closest.

**Reply by yhatt** [reply]
> You can start a new discussion from https://github.com/orgs/marp-team/discussions/new/choose.

Check out https://github.com/quarto-dev/quarto-cli/discussions/7377. We leave the decision whether to add the integration for Quatro to Quatro community.

**Reply by RoyiAvital** [reply]
@yhatt , The option to create a discussion is blocked for me:

![image](https://github.com/user-attachments/assets/4d791280-9b40-49e3-9d8f-28fc7b91f6df)

I guess it is limited to a sub set of users or something as I am able to open a thread on other communities.

**Reply by yhatt** [reply]
Thank you for details. I've recovered the creation of new discussions.
